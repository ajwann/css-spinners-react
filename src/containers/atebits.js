import React from 'react';
import './css/spinner/atebits.css';

const Atebits = props => {
  return(
    <div className="atebits-loader" {...props} />
  )
}

export { Atebits };
