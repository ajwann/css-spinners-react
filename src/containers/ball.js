import React from 'react';
import './css/spinner/ball.css';

const Ball = props => {
  return(
    <div className="ball-loader" {...props} />
  )
}

export { Ball };
