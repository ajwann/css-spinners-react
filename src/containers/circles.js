import React from 'react';
import './css/spinner/circles.css';

const Circles = props => {
  return(
    <div className="circles-loader" {...props} />
  )
}

export { Circles };
