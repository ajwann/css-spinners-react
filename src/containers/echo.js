import React from 'react';
import './css/spinner/echo.css';

const Echo = props => {
  return(
    <div className="echo-loader" {...props} />
  )
}

export { Echo };
