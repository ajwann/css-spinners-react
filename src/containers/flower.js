import React from 'react';
import './css/spinner/flower.css';

const Flower = props => {
  return(
    <div className="flower-loader" {...props} />
  )
}

export { Flower };
