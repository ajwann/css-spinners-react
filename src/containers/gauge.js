import React from 'react';
import './css/spinner/gauge.css';

const Gauge = props => {
  return(
    <div className="gauge-loader" {...props} />
  )
}

export { Gauge };
