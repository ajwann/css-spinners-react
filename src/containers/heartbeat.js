import React from 'react';
import './css/spinner/heartbeat.css';

const Heartbeat = props => {
  return(
    <div className="heartbeat-loader" {...props} />
  )
}

export { Heartbeat };
