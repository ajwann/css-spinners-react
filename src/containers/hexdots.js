import React from 'react';
import './css/spinner/hexdots.css';

const Hexdots = props => {
  return(
    <div className="hexdots-loader" {...props} />
  )
}

export { Hexdots };
