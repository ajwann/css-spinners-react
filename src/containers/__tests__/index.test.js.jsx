import React from 'react';
import renderer from 'react-test-renderer';

import { Atebits } from '../atebits.js';
import { Ball } from '../ball.js';
import { Circles } from '../circles.js';
import { Dots } from '../dots.js';
import { Echo } from '../echo.js';
import { Flower } from '../flower.js';
import { Gauge } from '../gauge.js';
import { Heartbeat } from '../heartbeat.js';
import { Hexdots } from '../hexdots.js';
import { HolePulse } from '../hole-pulse.js';
import { InnerCircles } from '../inner-circles.js';
import { Plus } from '../plus.js';
import { Pong } from '../pong.js';
import { Pulse } from '../pulse.js';
import { Refreshing } from '../refreshing.js';
import { Spinner } from '../spinner.js';
import { SpinningPixels } from '../spinning-pixels.js';
import { ThreeQuarters } from '../three-quarters.js';
import { Throbber } from '../throbber.js';
import { Timer } from '../timer.js';
import { Whirly } from '../whirly.js';
import { Wobblebar } from '../wobblebar.js';

describe('spinner components', () => {
  it('renders atebits', () => {
    const atebits = renderer.create(<Atebits />).toJSON();
    expect(atebits).toMatchSnapshot();
  });
  it('renders ball', () => {
    const ball = renderer.create(<Ball />).toJSON();
    expect(ball).toMatchSnapshot();
  });
  it('renders circles', () => {
    const circles = renderer.create(<Circles />).toJSON();
    expect(circles).toMatchSnapshot();
  });
  it('renders dots', () => {
    const dots = renderer.create(<Dots />).toJSON();
    expect(dots).toMatchSnapshot();
  });
  it('renders echo', () => {
    const echo = renderer.create(<Echo />).toJSON();
    expect(echo).toMatchSnapshot();
  });
  it('renders flower', () => {
    const flower = renderer.create(<Flower />).toJSON();
    expect(flower).toMatchSnapshot();
  });
  it('renders gauge', () => {
    const gauge = renderer.create(<Gauge />).toJSON();
    expect(gauge).toMatchSnapshot();
  });
  it('renders heartbeat', () => {
    const heartbeat = renderer.create(<Heartbeat />).toJSON();
    expect(heartbeat).toMatchSnapshot();
  });
  it('renders hexdots', () => {
    const hexdots = renderer.create(<Hexdots />).toJSON();
    expect(hexdots).toMatchSnapshot();
  });
  it('renders hole-pulse', () => {
    const holePulse = renderer.create(<HolePulse />).toJSON();
    expect(holePulse).toMatchSnapshot();
  });
  it('renders inner-circles', () => {
    const innerCircles = renderer.create(<InnerCircles />).toJSON();
    expect(innerCircles).toMatchSnapshot();
  });
  it('renders plus', () => {
    const plus = renderer.create(<Plus />).toJSON();
    expect(plus).toMatchSnapshot();
  });
  it('renders pong', () => {
    const pong = renderer.create(<Pong />).toJSON();
    expect(pong).toMatchSnapshot();
  });
  it('renders pulse', () => {
    const pulse = renderer.create(<Pulse />).toJSON();
    expect(pulse).toMatchSnapshot();
  });
  it('renders refreshing', () => {
    const refreshing = renderer.create(<Refreshing />).toJSON();
    expect(refreshing).toMatchSnapshot();
  });
  it('renders spinner', () => {
    const spinner = renderer.create(<Spinner />).toJSON();
    expect(spinner).toMatchSnapshot();
  });
  it('renders spinning-pixels', () => {
    const spinningPixels = renderer.create(<SpinningPixels />).toJSON();
    expect(spinningPixels).toMatchSnapshot();
  });
  it('renders three-quarters', () => {
    const threeQuarters = renderer.create(<ThreeQuarters />).toJSON();
    expect(threeQuarters).toMatchSnapshot();
  });
  it('renders throbber', () => {
    const throbber = renderer.create(<Throbber />).toJSON();
    expect(throbber).toMatchSnapshot();
  });
  it('renders timer', () => {
    const timer = renderer.create(<Timer />).toJSON();
    expect(timer).toMatchSnapshot();
  });
  it('renders whirly', () => {
    const whirly = renderer.create(<Whirly />).toJSON();
    expect(whirly).toMatchSnapshot();
  });
  it('renders wobblebar', () => {
    const wobblebar = renderer.create(<Wobblebar />).toJSON();
    expect(wobblebar).toMatchSnapshot();
  });
});
