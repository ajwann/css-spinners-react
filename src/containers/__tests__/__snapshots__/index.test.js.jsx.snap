// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`spinner components renders atebits 1`] = `
<div
  className="atebits-loader"
/>
`;

exports[`spinner components renders ball 1`] = `
<div
  className="ball-loader"
/>
`;

exports[`spinner components renders circles 1`] = `
<div
  className="circles-loader"
/>
`;

exports[`spinner components renders dots 1`] = `
<div
  className="dots-loader"
/>
`;

exports[`spinner components renders echo 1`] = `
<div
  className="echo-loader"
/>
`;

exports[`spinner components renders flower 1`] = `
<div
  className="flower-loader"
/>
`;

exports[`spinner components renders gauge 1`] = `
<div
  className="gauge-loader"
/>
`;

exports[`spinner components renders heartbeat 1`] = `
<div
  className="heartbeat-loader"
/>
`;

exports[`spinner components renders hexdots 1`] = `
<div
  className="hexdots-loader"
/>
`;

exports[`spinner components renders hole-pulse 1`] = `
<div
  className="hole-pulse-loader"
/>
`;

exports[`spinner components renders inner-circles 1`] = `
<div
  className="inner-circles-loader"
/>
`;

exports[`spinner components renders plus 1`] = `
<div
  className="plus-loader"
/>
`;

exports[`spinner components renders pong 1`] = `
<div
  className="pong-loader"
/>
`;

exports[`spinner components renders pulse 1`] = `
<div
  className="pulse-loader"
/>
`;

exports[`spinner components renders refreshing 1`] = `
<div
  className="refreshing-loader"
/>
`;

exports[`spinner components renders spinner 1`] = `
<div
  className="spinner-loader"
/>
`;

exports[`spinner components renders spinning-pixels 1`] = `
<div
  className="spinning-pixels-loader"
/>
`;

exports[`spinner components renders three-quarters 1`] = `
<div
  className="three-quarters-loader"
/>
`;

exports[`spinner components renders throbber 1`] = `
<div
  className="throbber-loader"
/>
`;

exports[`spinner components renders timer 1`] = `
<div
  className="timer-loader"
/>
`;

exports[`spinner components renders whirly 1`] = `
<div
  className="whirly-loader"
/>
`;

exports[`spinner components renders wobblebar 1`] = `
<div
  className="wobblebar-loader"
/>
`;
